examples/IRMQTTServer/IRMQTTServer.ino

```diff
@@ -2961,28 +2966,24 @@ bool sendClimate(const String topic_prefix, const bool retain,
     diff = true;
     success &= sendInt(topic_prefix + KEY_MODEL, next.model, retain);
   }
+  String mode_str = IRac::opmodeToString(next.mode);
+  mode_str.toLowerCase();
```

Comment:
Worth having the comment here too, explaining that mode str needs to be lowercase for HA & Google Assistant?

Reply:
Fair call. Done.